python/lsst/meas/modelfit/psf.py

```diff
@@ -105,18 +104,35 @@ def measure(self, measRecord, exposure):
         psf = exposure.getPsf()
         psfImage = psf.computeKernelImage(measRecord.getCentroid())
         psfShape = psf.computeShape(measRecord.getCentroid())
+        lastError = None
+        lastModel = None
         # Fit the first element in the sequence, using the PSFs moments to initialize the parameters
-        fitter, key = self.sequence[0]
-        lastResult = fitter.apply(psfImage, psfShape)
-        lastModel = fitter.getModel()
-        measRecord.set(key, lastResult)
         # For every other element in the fitting sequence, use the previous fit to initialize the parameters
-        for fitter, key in self.sequence[1:]:
-            initial = fitter.adapt(lastResult, lastModel)
-            lastResult = fitter.apply(psfImage, initial)
-            lastModel = fitter.getModel()
-            measRecord.set(key, lastResult)
-
+        for fitter, name in self.sequence:
```

Comment:
I was just noting that they no longer refer to the lines immediately below them, but I guess they still make sense.